src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.targets

```diff
@@ -550,6 +550,11 @@ Copyright (C) 2011-2012 Xamarin. All rights reserved.
       ResourceName="XA1035"
       Condition=" '$(BundleAssemblies)' == 'true' and '$(UsingAndroidNETSdk)' == 'true' "
   />
+  <AndroidError Code="XA1036"
+      ResourceName="XA1036"
+      Condition="$([MSBuild]::VersionLessThan('$(SupportedOSPlatformVersion)', '$(AndroidMinimumSupportedApiLevel)'))"
```

Comment:
Still confused: how does this work?  Where is `$(AndroidMinimumSupportedApiLevel)` defined?  (How are the unit tests passing?!  Which strongly implies that this *works*, I just don't understand *how*.)

Reply:
See https://github.com/xamarin/xamarin-android/pull/8026/files#r1199292341